src/main/java/androidx/graphics/Bitmap.kt

```diff
@@ -118,17 +119,18 @@ inline fun createBitmap(
 }
 
 /**
- * Returns ByteArray compressed from this bitmap with the specified [format]
+ * Returns ByteArrayInputStream compressed from this bitmap with the specified [format]
  * and [quality].
  *
  * @param format The format of bitmap.
  * @param quality Hint to the compressor, 0-100. 0 meaning compress for small size, 100 meaning compress for max quality.
- * @return ByteArray
+ * @return ByteArrayInputStream
  */
-inline fun Bitmap.toByteArray(
+inline fun Bitmap.toStream(
```

Comment:
That is not what I meant. What I meant is that there shouldn't be an extension on `Bitmap` to compress to a byte array or a stream, etc. `Bitmap.compress()` already accepts an `OuputStream` which is generic enough to cover all use cases. It shouldn't be specialized here.